Dockerfile

```diff
@@ -1,20 +1,22 @@
-FROM node:16-bullseye
+FROM node:16-alpine as ts-builder
 
-WORKDIR /home/backend-skopos
+WORKDIR /home/backend_skopos
 
+COPY . ./
 
-COPY package*.json ./
-
-ENV PRISMA_SKIP_POSTINSTALL_GENERATE=true
+RUN npm run clean
+RUN npm run build 
 
-RUN npm install #--loglevel verbose
 
-COPY . .
+FROM node:16-alpine as ts-prod
+WORKDIR /home/backend_skopos
 
-RUN npx prisma generate
-RUN npm run build 
+COPY --from=ts-builder ./home/backend_skopos/dist ./dist
+COPY --from=ts-builder ./home/backend_skopos/prisma ./prisma
+COPY package* ./
 
+RUN npm install --production
 
 EXPOSE 3001
 
-CMD ["npm", "run", "deploy"]
+CMD npx prisma migrate deploy; node ./dist/index.js
```

Comment:
Is there a reason for not running the deploy command?

Reply:
npm run deploy was failing in the container; if it works for you, no need to change